.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "rust-analyzer.cargo.features": [
-        "simd-stable"
+        "simd-stable",
+        "dim3"
```

Comment:
this is not needed, but not an entirely bad thing :thinking: ; should we ignore this file in gitignore though ?

Reply:
Yes, we can remove/ignore this file.